docker-compose.yml

```diff
@@ -1,12 +1,21 @@
 version: '3'
 services:
+
     web:
-        build: ./
-        container_name: svgtranslate_web
+        build:
+            dockerfile: ./.docker/web/Dockerfile
+            context: .
         ports:
-          - ${TOOLFORGE_DOCKER_PORT}:80
+            - ${TOOLFORGE_DOCKER_PORT}:80
         volumes:
-          - ./.docker/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
-          - ./:/var/www:cached
+            - ./:/var/www:cached
         environment:
-          TOOLFORGE_DOCKER_PORT: ${TOOLFORGE_DOCKER_PORT}
+            TOOLFORGE_DOCKER_PORT: ${TOOLFORGE_DOCKER_PORT}
+
+    assets:
+        build:
+            dockerfile: ./.docker/assets/Dockerfile
+            context: .
+        volumes:
+            - ./:/svgtranslate:cached
+        command: tail -F /dev/null
```

Comment:
Do you still want it to run `npm install` or no?

Reply:
I do, but adding that as the command makes it stop the container after npm install is complete, and I want the developer to be able to later run other commands (build assets etc.). Is there a way to do that? For now, I've fixed it by keeping the `assets/run.sh` script.